QUEST.tsv

```diff
@@ -2098,156 +2098,156 @@ QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion t
-QUEST_20150714_002228	Thanks.{nl}If you see any bubble that spurts out air, it means that there are spiney water chestnuts.{nl}Please bring me the roots of them.
-QUEST_20150714_002229	Really?{nl}If you could help me, I will do anything I can.
-QUEST_20150714_002230	We retrieved the hall.{nl}Let's keep moving forward!
-QUEST_20150714_002231	Let's save the chief fast!
-QUEST_20150714_002232	Finally, the time to retrieve the monastery has come.
-QUEST_20150714_002233	I will go with the revelator.{nl}Let's do it well.
-QUEST_20150714_002234	Let's run fast from the next corridor.{nl}I know you are exhausted, but we are almost there.
-QUEST_20150714_002235	I will block the monsters that are rushing in here.{nl}You and Gus move up first.
-QUEST_20150714_002236	Thanks.{nl}I saw Moheim brothers entering inside the monastery a while ago.{nl}They are also responsible for keeping the important objects, so you better ask them.
-QUEST_20150714_002237	I am sorry that I bothered you.{nl}The boxes are lined up inside the reception room.
-QUEST_20150714_002238	So you've come.{nl}Please give me the Crystal Ball.
-QUEST_20150714_002239	Damn! Until the end…{nl}You two follow that guy and the rest of you help me!
-QUEST_20150714_002240	The hunter who is caring for his colleagues
-QUEST_20150714_002241	Hey…! Hold yourself!
-QUEST_20150714_002242	Thanks. I have too many tasks to do…{nl}I gotta sort out the green cover pages, but I can't find any.{nl}Please look for them from the furnitures inside the monastery.
-QUEST_20150714_002243	For what purpose did you visit this cursed city?{nl}Please go back if you don't have a reason for being here.
-QUEST_20150714_002244	So you are the famous Revelator. {nl}I definitely heard about you before.{nl}
-QUEST_20150714_002245	I will permit your entrance temporarily. {nl}Caliss knightage is looking for competent people like you.
-QUEST_20150714_002246	But, all the authority here belongs to our kingdom military.{nl}I hope you behave well.
-QUEST_20150714_002247	Saltistter? What are you talking about?{nl}You are saying such a non-sense…
-QUEST_20150715_002248	Modestas accepted his defeat and promised his pay back later. When Modestas was about to leave with his horse and wagon, the old Alchemist with a grin took Modesta's attention and made him look at him. {nl}
-QUEST_20150715_002249	Modestas accepted his defeat and promised his pay back later. When Modestas was about to leave with his horse and wagon, the old Alchemist with a grin took Modesta's attention and made him look at him. {nl}
-QUEST_20150717_002250	Please insert the name of your companion.
+QUEST_20150714_002101	$Hones, who is one of them, is lacking in scholarly intelligence. However, I am sure his faithfulness and efforts will help him follow my teachings.
+QUEST_20150714_002102	$The Records of the Hidden Object
+QUEST_20150714_002103	$…I don't know whether or not I should thank the Goddesses for this discovery. My greed for learning can ruin the world.
```

Comment:
I'd move "or not" to the end of the sentence. Also, replace "learning" with "knowledge"